File: nodejs/idealo.js
const request = require('request');

const username = 'YOUR_USERNAME';
const password = 'YOUR_PASSWORD';

const options = {
  method: 'POST',
  url: 'https://scrape.smartproxy.com/v1/tasks',
  headers: {
    'Content-Type': 'application/json',
    'Authorization': 'Basic ' + Buffer.from(username + ':' + password).toString('base64')
  },
  body: JSON.stringify({
    target: 'idealo',
    parse: false,
    url: 'https://www.idealo.co.uk/compare/200584791/sony-playstation-5-ps5-digital-edition.html'
  })
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: python/idealo.py
import requests

task_params = {
    'target': 'idealo',
    'url': 'https://www.idealo.co.uk/compare/200584791/sony-playstation-5-ps5-digital-edition.html',
    'parse': False,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scrape.smartproxy.com/v1/tasks',
    json = task_params,
    auth = (username, password)
)

print(response.text)